### ISSUE TITLE
Aggressively removes images

### ISSUE THREAD

**Issue by rupeshk** [buriy/python-readability#89]
Between 0.3.0.6 and current release python-readability aggressively removes all images embedded in the html. There doesn't seem to be a way to control this behaviour.

**Comment by buriy** [comment]
Please suggest what do you like to control specifically.

**Comment by baby5** [comment]
@buriy he wants to keep img tag but there seem no way

**Comment by buriy** [comment]
So, the idea is: Add an option to keep all images.

**Comment by Uzay-G** [comment]
If I open a PR for this could it be merged?

**Comment by buriy** [comment]
Yes, you can add this option (to keep all images) and make a PR.

**Comment by buriy** [comment]
Thanks, resolved with an option to keep all images but the main one.
